Automattic/sensei
# Update footer styles in multi page quiz

Resolves https://github.com/Automattic/sensei/issues/7243

## Proposed Changes
* Updated styles for multi-paged quizzes in learning mode

## Testing Instructions


1. Add a multi-page quiz to a lesson.
2. Select the Allow Retakes checkbox.
3. Checkout this branch of Course theme https://github.com/Automattic/themes/pull/7462
4. As a student, view the quiz on the frontend.
5. Scroll to the footer.
6. Make sure the design matches the new structure of footers across LM
7. Make sure normal single page footer is not broken
8. Make sure lesson footer is not broken
9. Check in mobile view
10. Check for all variations of course theme and other top used themes

#### Course theme
![Screenshot 2023-11-06 at 3 55 49 PM](https://github.com/Automattic/sensei/assets/6820724/b0992cb5-f251-4970-8ead-37db9223a6ee)
![Screenshot 2023-11-06 at 3 56 20 PM](https://github.com/Automattic/sensei/assets/6820724/058e58c3-0e07-4093-8ab5-97d5ed9c315d)

#### Astra
![Screenshot 2023-11-06 at 3 56 48 PM](https://github.com/Automattic/sensei/assets/6820724/1e4574a4-7eac-445f-8eb1-502dba8fcc73)

## Pre-Merge Checklist
